apis/Google.Cloud.Datastore.V1/Google.Cloud.Datastore.V1.IntegrationTests/DatastoreDbTest.cs

```diff
@@ -441,17 +442,192 @@ public void Count_WithStructuredQuery()
             var query = new Query("CountTestStQuery");
             AggregationQuery aggQuery = new AggregationQuery(query)
             {
-                Aggregations = { Aggregations.Count("count")}
+                Aggregations = { Count("count")}
             };
             AggregationQueryResults results = db.RunAggregationQuery(aggQuery);
             long count = results["count"].IntegerValue;
             Assert.Equal(2, count);
         }
 
         [Fact]
-        public void OrQueries()
+        public void Sum_WithGqlQuery()
```

Comment:
I really don't think we need all of these tests. I would create *one* test for a structured query, and *one* test for GQL, specifying two sums and an average with some mixture of aliased and not aliased. It's just a general integration test to make sure everything basically works... we don't need to be exhaustive, given that most of what's being tested is the server rather than the client.